docs/designers-developers/developers/backward-compatibility/README.md

```diff
@@ -1 +1,57 @@
 # Backward Compatibility
+
+Gutenberg follows the WordPress backwards compatibility principles. This means Backward compatibility for producation public API is guaranteed. In some rare occasions breaking backward compatibility is unavoidable, in that case, the breakage should be as small as possible and the breakage should be documented as clearly as possible to third-party developpers using devnotes.
```

Comment:
I see `backwards compatibility` and `Backward compatibility`.

Reply:
What do you think I should use?

Reply:
```suggestion
Gutenberg follows the WordPress backward compatibility principles. This means backward compatibility for production public API is guaranteed. In some rare occasions breaking backward compatibility is unavoidable, in that case, the breakage should be as small as possible and be documented as clearly as possible to third-party developers using dev-notes.
```